setup.py

```diff
@@ -32,8 +32,11 @@
     include_package_data=True,
     test_suite='test_bootstrap_flask',
     install_requires=[
-        'Flask'
+        'Flask>=1'
```

Comment:
Why need Flask **>= 1**?

Reply:
Because `Flask` versions below `v1.0.0` require `Jinja2>=2.4`, while we need at least `Jinja2>=2.8` (Or around there). Maybe it would be better to add an explicit requirement?

Reply:
All right...